frontend/projects/ui/src/app/app.component.ts

```diff
@@ -133,19 +107,15 @@ export class AppComponent {
             ])
           })
         // UNVERIFIED
-      } else if (auth === AuthState.UNVERIFIED) {
+      } else {
```

Comment:
:speech_balloon: This is fine, since the type is bool. I tend not to like bools, but not seeing the downside here.